lint.sh

```diff
@@ -17,6 +17,11 @@
 function lintAllFiles () {
   echo "Running linter on module $1"
   pylint --disable=$2 $1
+  rc=$?
```

Comment:
Done. I added some more argument validation to make it work better with `nounset` option.